vm/page_pool_alloc/src/lib.rs

```diff
@@ -288,6 +286,8 @@ struct PagePoolInner {
     /// The list of device ids for outstanding allocators. Each name must be
     /// unique.
     device_ids: Vec<DeviceId>,
+    /// The mapper used to create mappings for allocations.
+    mapper: Box<dyn Mapper>, // todo inspect?
```

Comment:
inspect seems useful, yes

Reply:
I guess it would indeed be useful to know what kind of mapper you have eh?